src/renderers/webgl/WebGLTextures.js

```diff
@@ -410,7 +410,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 							for ( var j = 0; j < mipmaps.length; ++ j ) {
 
 								var mipmap = mipmaps[ j ];
-								var image = mipmap.image[ i ].image;
+								image = mipmap.image[ i ].image;
```

Comment:
Uh oh, I would call this one `mipmapImage`.

Reply:
👍